desk/lib/bolt.hoon

```diff
@@ -39,9 +38,7 @@
   +*  this  .
       ag    ~(. yosh bowl)
       io    ~(. agentio bowl)
-      perms  ~(. permissions bowl state)
-      allowed  %:(is-allowed:perms src.bowl)
-      emit   ~(website json state)
```

Comment:
Same thing here, `json` has access to `state` since you pinned it before the `=<`. Whether you want to force yourself to pass it as an argument or not is certainly up for debate (I myself go back and forth and don't claim consistency) but at least use a consistent layout/architecture throughout each file. :)